src/main/java/brokeculator/command/HelpCommand.java

```diff
@@ -7,13 +7,25 @@ public class HelpCommand extends Command {
     private static final String HELP_MESSAGE = "Here are the commands you can use:" + System.lineSeparator()
             + "\t1. add /n <description> /d <date> /a <amount> [/c <category>] - adds an expense to the dashboard"
             + System.lineSeparator()
-            + "\t2. delete <index> - deletes an expense from the dashboard" + System.lineSeparator()
-            + "\t3. list - lists all expenses in the dashboard" + System.lineSeparator()
+            + "\t2. delete <index> - deletes an expense from the dashboard"
+            + System.lineSeparator()
+            + "\t3. list - lists all expenses in the dashboard"
+            + System.lineSeparator()
             + "\t4. summarise [/from <start>] [/to <end>] - summarises the expenses in the dashboard"
             + System.lineSeparator()
-            + "\t6. category <list|add|delete> [<category if add|delete specified>] - manages categories"
+            + "\t5. category <list|add|delete> [<category if add|delete specified>] - manages categories"
+            + System.lineSeparator()
+            + "\t6. event /n <event name> /d <event description> - adds an event"
+            + System.lineSeparator()
+            + "\t7. viewEvent /i <index> - views an event"
+            + System.lineSeparator()
+            + "\t8. viewEvents - lists all events"
+            + System.lineSeparator()
+            + "\t9. deleteEvent /i <index> - deletes an event" 
+            + System.lineSeparator()
+            + "\t10. addExpenseToEvent /exi <expense index> /evi <event index> - adds an expense to an event"
```

Comment:
Perhaps add a command to delete an expense from an event and a command to summarise expenses in an event?

Reply:
Delete command added